docs/resources/privatelink.md

```diff
@@ -80,6 +80,10 @@ terraform import risingwavecloud_privatelink.test <privatelink_id>
 - `connection_name` (String) The name of the Private Link connection, just for display purpose.
 - `target` (String) The target of the Private Link connection. In AWS, it is the service name of the VPC endpoint service. In GCP, it is the service attachment in Private Service Connect.
 
+### Optional
+
+- `wait_for_ready` (Boolean) Whether to wait for the private link to be ready after creation. Default is false.
```

Comment:
There are two states for a private link, its [lifecycle status](https://github.com/risingwavelabs/risingwave-cloud/blob/046900c78641a7a85baeb961427c73314bc212f8/mgmt/internal/model/privatelink.go#L18-L22) & its [connection state](https://github.com/risingwavelabs/risingwave-cloud/blob/046900c78641a7a85baeb961427c73314bc212f8/mgmt/internal/infra/network/types.go#L45-L49)

What does it mean by "ready" here?

Reply:
It is `CREATED`.

Reply:
Hmm, I previously had some misunderstanding then -- it makes sense to me to not wait for connection state to turn `ACCEPTED`(which is my original understanding) but what's the use case for not waiting for a privatelink to be `CREATED`? 

As an analogy, do we want to skip waiting for a RW tenant to turn `CREATED`? All resources have a similar lifecycle status pattern, I feel we should be consistent among them.

Reply:
make sense, just make it this way